ygot/struct_validation_map.go

```diff
@@ -63,6 +63,11 @@ func structTagToLibPaths(f reflect.StructField, parentPath *gnmiPath) ([]*gnmiPa
 			}
 			ePath.AppendName(pp)
 		}
+
+		if len(p) > 0 && p[0] == '/' {
+			ePath.isAbsolute = true
+		}
+
```

Comment:
if you were to use isAbsolute() method, you could omit this and skip only if idx != 0 in the for loop above.

Reply:
Agreed -- see above.